schemas/context/opportunity-crm.schema.json

```diff
@@ -0,0 +1,136 @@
+{
+  "meta:license": [
+    "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/opportunity-crm",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "CRM Opportunity",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:intendedToExtend": [
+    "https://ns.adobe.com/xdm/context/opportunity"
+  ],
+  "description": "CRM specific opportunity fields.",
+  "definitions": {
+    "opportunity-crm": {
+      "properties": {
+        "xdm:name": {
+          "title": "Opportunity Name",
+          "description": "Subject or descriptive name, such as the expected order or company name, for the opportunity.",
+          "type": "string"
+        },
+        "xdm:description": {
+          "title": "Opportunity Description",
+          "description": "Additional information to describe the opportunity, such as possible products to sell or past purchases from the customer.",
+          "type": "string"
+        },
+        "xdm:stage": {
+          "title": "Opportunity Stage",
+          "description": "Sales stage of this opportunity to aid the sales team in their efforts to win this opportunity.",
+          "type": "string"
+        },
+        "xdm:amount": {
+          "title": "Opportunity Amount",
+          "description": "Opportunity Amount.",
+          "type": "number"
+        },
+        "xdm:quantity": {
+          "title": "Opportunity Quantity",
+          "description": "Opportunity Quantity.",
+          "type": "number"
+        },
+        "xdm:fiscalYear": {
+          "title": "Fiscal Year",
+          "description": "Fiscal Year this opportunity is targeted.",
+          "type": "string"
+        },
+        "xdm:fiscalQuarter": {
+          "title": "Fiscal Quarter",
+          "description": "Fiscal Quarter this opportunity is targeted..",
+          "type": "string"
```

Comment:
Should we consider enums such as Q1 - Q4?

Reply:
I have seen customers using numbers like 1,2,3,4 and strings like Q1-Q4 . Not sure if it is good idea to limit only to Q1-Q4

Reply:
Check if JSON Schema validation allows a "OR" during pattern match?